src/components/PsaBanner/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { FEATURES } from '@gnosis.pm/safe-react-gateway-sdk'
+import { useSelector } from 'react-redux'
+import Close from '@material-ui/icons/Close'
+import { currentChainId } from 'src/logic/config/store/selectors'
+import { hasFeature } from 'src/logic/safe/utils/safeVersion'
+import useCachedState from 'src/utils/storage/useCachedState'
+import { useEffect, useState } from 'react'
+import { loadFromCookie } from 'src/logic/cookies/utils'
+import { COOKIES_KEY, BannerCookiesType } from 'src/logic/cookies/model/cookie'
+
+const BANNERS = {
+  '4': <>🚨 Rinkeby is being deprecated. Please migrate to Goerli. 🚨</>,
+  '5': (
+    <>
+      🚨 On <b>13.10.2022 @ 9am CEST</b> we will be undertaking indexer maintenance on Goerli, during which the
+      functionality of this application might be restricted Please expect downtime of 2-3 hours. 🚨
```

Comment:
I think you meant to add a full stop after "restricted", for all of these banners.